src/components/categories/CategoryList.js

```diff
@@ -14,11 +29,24 @@ export default (props) => {
     <>
       <article>
         <h2 className="my-4 text-center">Existing Categories</h2>
-        <Row className="justify-content-center">
+        <Container fluid>
+
           {categories.map((c) => (
+            <>
+            <Row className="">
```

Comment:
There is a react bootstrap element called ListGroupItem that would look good here. It would add horizontal rows, which would help the user see which buttons go with which tags.